### PR TITLE
Sechub sambr

### PR DESCRIPTION
**Pull request by samzsam**
### Related ticket(s)

MDCT-

---
### Important updates

---
### Author checklist

- [x] I have performed a self-review of my code
- [x] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [x] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

**Comment by samzsam** [comment]
Added implementation of AWS SecurityHub findings to JIRA
Removed previous implementation of SecurityHub findings to GitHub issues

File: .github/workflows/schedule-jira-sync.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+      - name: Sync Security Hub and Jira
+        uses: Enterprise-CMCS/security-hub-visibility@v1.0.1
+        with:
+          jira-token: ${{ secrets.JIRA_SERVICE_USER_TOKEN }}
+          jira-username: ${{ secrets.JIRA_SERVICE_USERNAME }}
+          jira-host: qmacbis.atlassian.net
+          jira-project-key: MDCT 
+          jira-epic-key: MDCT-2280 
+          jira-ignore-statuses: Done
```
Comment (braxex) [inline .github/workflows/schedule-jira-sync.yml:33]: Question: We also use "Canceled" and "Closed". Should we add those?

**Review by tblackwe — DISMISSED** [review]
LGTM

File: .github/workflows/schedule-jira-sync.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+permissions:
+  id-token: write
+
+jobs:
+  sync:
+    name: Run sync
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v3
```
Comment (benmartin-coforma) [inline .github/workflows/schedule-jira-sync.yml:17]: Question: It doesn't seem like we're using any files from the repo in latter steps. Can we remove this one?

File: .github/workflows/schedule-jira-sync.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        uses: Enterprise-CMCS/security-hub-visibility@v1.0.1
+        with:
+          jira-token: ${{ secrets.JIRA_SERVICE_USER_TOKEN }}
+          jira-username: ${{ secrets.JIRA_SERVICE_USERNAME }}
+          jira-host: qmacbis.atlassian.net
+          jira-project-key: MDCT 
+          jira-epic-key: MDCT-2280 
+          jira-ignore-statuses: Done
+          jira-custom-fields: '{ "application" : "MCR" }'
+          aws-region: us-east-1
```
Comment (benmartin-coforma) [inline .github/workflows/schedule-jira-sync.yml:35]: Suggestion: Use `${{ secrets.AWS_DEFAULT_REGION }}`, to match the Configure Credentials step.

File: scripts/deploy.sh
Comment (benmartin-coforma) [inline scripts/deploy.sh:1]: Something weird's going on with this file. It looks normal when I pull this branch down & view on my machine, but github is showing it as empty in this diff. I don't think this is blocking; I'm just confused.

File: services/.sechub/handlers/sync.js
Comment (benmartin-coforma) [inline services/.sechub/handlers/sync.js:1]: Woohoo! Goodbye `sync.js`. Thank you @samzsam !

**Review by braxex — APPROVED** [review]
Approved once the conversations are closed.

File: .github/workflows/schedule-jira-sync.yml (hunk 1 above)
Comment (samzsam) [reply .github/workflows/schedule-jira-sync.yml:33]: yes comma-separated list of Jira issue statuses that will be ignored when syncing can be added

Comment (braxex) [reply .github/workflows/schedule-jira-sync.yml:33]: @samzsam Great! Will you add those so we have Done, Canceled, and Closed for all the PRs?

File: .github/workflows/schedule-jira-sync.yml (hunk 3 above)
Comment (samzsam) [reply .github/workflows/schedule-jira-sync.yml:35]: yes it can be removed

File: .github/workflows/schedule-jira-sync.yml (hunk 2 above)
Comment (samzsam) [reply .github/workflows/schedule-jira-sync.yml:17]: I suggest to keep it just incase.

File: services/.sechub/handlers/sync.js
Comment (samzsam) [reply services/.sechub/handlers/sync.js:1]: Thank you!

**Review by braxex — APPROVED** [review]

**Pull request merged**